suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -719,26 +452,40 @@ def get_xlsx(data: Any, args: ArgparseNamespace) -> BytesIO:  # noqa: ANN401
 
     # the columns that should be styled as required (grey background)
     required_columns = [
-        "Titel",
-        "Komponist",
-        "Interpret",
+        "Sender",
+        "Titel des Musikwerks",
+        "Name des Komponisten",
+        "Interpret(en)",
         "Sendedatum",
         "Sendedauer",
         "Sendezeit",
         "ISRC",
         "Label",
-        "Label Code",
         "Identifikationsnummer",
+        "Eigenaufnahmen",
+    ]
+    subsidiary_columns = [
+        "EAN/GTIN",
+        "Albumtitel / Titel des Tonträgers",
+        "Aufnahmedatum",
+        "Aufnahmeland",
+        "Erstveröffentichungsdatum",
+        "Katalog-Nummer / CD ID",
+        "Werkverzeichnisangaben",
+        "Bestellnummer",
     ]
     font = Font(name="Calibri", bold=True, size=12)
     side = Side(border_style="thick", color="000000")
     border = Border(top=side, left=side, right=side, bottom=side)
-    fill = PatternFill("solid", bgColor="d9d9d9", fgColor="d9d9d9")
+    required_fill = PatternFill("solid", bgColor="d9d9d9", fgColor="d9d9d9")
+    subsdiary_fill = PatternFill("solid", bgColor="ebf1de", fgColor="ebf1de")
```

Comment:
i checked the inofficial original for that actual color code (instead of using the eyedropper on the PDF):
```suggestion
    subsdiary_fill = PatternFill("solid", bgColor="aef0dd", fgColor="aef0dd")
```

Reply:
> i checked the inofficial original for that actual color code (instead of using the eyedropper on the PDF):

What is the "unofficial original"?

I compared and took the colors from the officially published [Gemeinsamer Tarif S 2026 – 2028](https://www.suisa.ch/dam/jcr:ea035e73-ce1a-48b7-9269-3f6aedc4057f/GTS_2026-2028_GER.pdf) document. There it's `bfbfbf` for the mandatory columns (_grey_, "zwingend") and `ebf1de` for the mandatory if no ISRC columns (_green_, zwingend, falls kein ISRC):

<img width="817" height="243" alt="image" src="https://github.com/user-attachments/assets/c28da166-bb93-45b9-9e5f-36ac82cd57ba" />